src/DSharp.Compiler.Tests/Source/Expression/ScriptIgnore/Baseline.txt

```diff
@@ -0,0 +1,27 @@
+﻿"use strict";
+
+define('test', ['ss'], function(ss) {
+  var $global = this;
+  // ExpressionTests.Program
+
+  function Program() {
+  }
+  var Program$ = {
+    main: function() {
+      var e1 = ss.enumerate(this);
+      var e2 = ss.enumerate((this));
```

Comment:
Interesting that it double braces here.

Reply:
it's because of the parenthesis surrounding the cast to be able to call the method `((X)b).a()` => `(b).a()` => `ss.a((b))`